content/ecosystem/integrations/integrations.toml

```diff
@@ -204,4 +204,15 @@ maintainer = "etke.cc"
 language = "Python"
 licence = "AGPL-3.0-only"
 repository = "https://gitlab.com/etke.cc/radicale-auth-matrix"
-room = "#source:etke.cc"
\ No newline at end of file
+room = "#source:etke.cc"
+
+[[integrations]]
+name = "Draupnir"
+description = """
+A moderation tool for Matrix.
+"""
+maintainer = "Gnuxie"
+language = "TypeScript"
+licence = "COOPERATIVE SOFTWARE LICENSE"
```

Comment:
Not sure how else to do it. This has no spdx identifier. So I took the title from the LICENSE file

Reply:
It has just been relicensed as AFL-3.0